Python/bltinmodule.c

```diff
@@ -2517,8 +2517,8 @@ builtin_issubclass_impl(PyObject *module, PyObject *cls,
 
 typedef struct {
     PyObject_HEAD
-    Py_ssize_t          tuplesize;
-    PyObject *ittuple;                  /* tuple of iterators */
+    Py_ssize_t tuplesize;  /* negative when strict=True */
```

Comment:
Why not using a separated member for strict? I expected "int strict".

Reply:
Agreed. Encoding an extra bit in the sign is so 1975. :-)

Reply:
But this encoding (size + strict) makes sense for getstate/setstate: it provides backward compatibility for pickle which is nice!

Reply:
I think I was feeling a bit too clever here... and preempting the argument that I was bloating `zip` instances. ;)

Changes coming soon.